cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -65,12 +65,6 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
         ).file(mustExist = true, canBeFile = true, canBeDir = true, mustBeWritable = false, mustBeReadable = true)
     ).single().required()
 
-    private val scopesToScan by option(
```

Comment:
Some misc. comments:
- Ideally, the new feature to ignore all excluded packages in a scan would be implemented first, so the rationale becomes less hypothetical and the justification for the removal more apparent. But for the sake of first reducing complexity instead of temporarily increasing it I'm fine with doing it in this order.
- Still, I'd prefer to the commit message body to be more concrete, e.g. "An upcoming change will introduce an option to ignore all packages from the scan which are excluded via the repository configuration. As scopes can be excluded via a ScopeExclude, that upcoming feature will superseded the dedicated command line option. Remove the so far unused option already now to reduce code complexity."
- I don't really get the last paragraph of your commit message, but I for me my above proposal would be sufficient.

Reply:
I changed the message and hope it addresses all your above points.

Reply:
TBH, I still don't like the opening "It seems to make sense to provide an option" part. It sounds as if you're unsure what you're going to do. And if you're not sure that the upcoming feature makes sense, then you must not remove this code.

In "Since such an fully covers" I guess the word "option" is missing.

Reply:
Done.